test/ProgressPlugin.test.js

```diff
@@ -79,6 +79,7 @@ const RunCompilerAsync = compiler =>
 	});
 
 describe("ProgressPlugin", function () {
+	jest.setTimeout(10000);
```

Comment:
Should work without this...

Reply:
reverted

Reply:
failed randomly again in ProgressPlugin
this seems ci error...